src/ext-test/java/org/opentripplanner/ext/siri/SiriEtBuilder.java

```diff
@@ -210,15 +248,21 @@ public EstimatedCallsBuilder call(StopLocation stop) {
 
     public EstimatedCallsBuilder arriveAimedExpected(String aimedTime, String expectedTime) {
       var call = calls.getLast();
-      call.setAimedArrivalTime(dateTimeHelper.zonedDateTime(aimedTime));
-      call.setExpectedArrivalTime(dateTimeHelper.zonedDateTime(expectedTime));
+      call.setAimedArrivalTime(aimedTime == null ? null : dateTimeHelper.zonedDateTime(aimedTime));
+      call.setExpectedArrivalTime(
+        expectedTime == null ? null : dateTimeHelper.zonedDateTime(expectedTime)
```

Comment:
Can you add a method to the date time helper to extract the repeated null checks?